modules/cloudsql-postgres/variables.tf

```diff
@@ -0,0 +1,151 @@
+variable "name" {
+  description = "Cloud SQL instance name (lowercase letters, numbers, and hyphens; up to 98 characters)."
+  type        = string
+}
+
+variable "project" {
+  description = "GCP project ID hosting the Cloud SQL instance."
+  type        = string
+}
+
+variable "region" {
+  description = "GCP region for the primary instance."
+  type        = string
+}
+
+variable "network" {
+  description = "Self‑link or name of the VPC network used for private IP connectivity."
+  type        = string
+}
+
+variable "require_squad" {
+  description = "Whether the `squad` variable must be specified."
+  type        = bool
+  default     = true
+}
```

Comment:
we should drop this and make it require

we only added this option for existing modules
new modules should just require it